src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohort.java

```diff
@@ -36,7 +36,7 @@
     @Argument(
             fullName = "cohort-extract-table",
             doc = "Fully qualified name of the table where the cohort data exists (already subsetted)",
-            optional = false
+            optional = true
```

Comment:
is this change needed for the case where you're using an avro file directly rather than pulling from bq? and did we come up with any use cases for this other than ease of testing?

Reply:
what if we keep this as `optional = false`, and then change the logic in ExtractCohortEngine from `if (cohortTableRef != null){` to `if (cohortAvroFileName == null)` ? 

then in your test you can provide a fake cohort extract table name, but it'll still read from your avro file.

Reply:
these comments are moot now